docs/getting-started/runner.md

```diff
@@ -3,9 +3,11 @@ title: Runner
 sidebar_position: 6
 ---
 
-## A developer tool and a DApp for running contracts on the blockchain
+In this video, Nick Stanford, Developer Relations Manager for Smart Contracts, shows you how to deploy and execute contracts created in Marlowe Playground using Marlowe Runner, beta version. Marlowe Runner is a developer tool with a friendly user interface for deploying and executing smart contracts on Cardano, straight from a browser.
```

Comment:
It is easier to keep the author of the video within YouTube rather than embed this within the docs

Reply:
@sfoo-iohk What do you suggest as an alternative? We have a similar page for Playground that mentions Pablo by name. 
https://docs.marlowe.iohk.io/docs/developer-tools/playground

Reply:
I don't have a big preference for this one. Both options are fine. But perhaps it's best to keep documentation general, eg: 'This video explains how to deploy and execute...'
Also, just a minor note: 'Lower case for job titles, degree names, and university departments' according to the style guide.